### PR TITLE
Do not throw error in `NetMQTransport.DoBroadcast()` method & release 0.25.3

### PR DESCRIPTION
**Pull request by riemannulus**
The subject says it all

### CODE REVIEW

**Comment by longfin** [comment]
I think it would be good we have a changelog like...

> # Bug fixes
> - Fixed a bug when message broadcasting on `Swarm<T>` had been stopped suddenly.

**Comment by riemannulus** [comment]
> I think it would be good we have a changelog like...
> 
> > # Bug fixes
> > 
> > * Fixed a bug when message broadcasting on `Swarm<T>` had been stopped suddenly.

Ok, I missed it.

**Comment by riemannulus** [comment]
@longfin I merge with 0.25.3 release on here. PTAL!

**Review by longfin — DISMISSED** [review]

File: CHANGES.md [hunk 1]
```diff
@@ -4,7 +4,14 @@ Libplanet changelog
 Version 0.25.3
 --------------
 
-To be released.
+Released on January 14, 2022
+
+### Bug Fixes
+
```
Comment (dahlia) [inline CHANGES.md:10]: Patch releases don't need sections as they are only for bug fixes.

```suggestion
```

File: CHANGES.md [hunk 2]
```diff
@@ -4,7 +4,14 @@ Libplanet changelog
 Version 0.25.3
 --------------
 
-To be released.
+Released on January 14, 2022
```
Comment (dahlia) [inline CHANGES.md:7]:
```suggestion
Released on January 14, 2022.
```

A trivial typo.

**Review by longfin — APPROVED** [review]

**Review by dahlia — APPROVED** [review]

**Pull request merged**